tests/pylorax/test_projects.py

```diff
@@ -330,7 +330,7 @@ def singlerepo_v0():
         "check_gpg": True,
         "check_ssl": True,
         "gpgkey_urls": [
-            "file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-32-x86_64"
+            "file:///etc/pki/rpm-gpg/RPM-GPG-KEY-" + os.environ['TEST_OS'] + "-x86_64"
```

Comment:
That only works if TEST_OS is defined, which in the case of running it with podman, it won't.
I'm also not sure that the key name will always match the TEST_OS value (eg. RHEL, CentOS)

Reply:
Well I just tried running it in podman and those blueprints tests are skipped there (SKIP: Test requires a running API server). I don't know if this is the correct behavior but it seems that this will not cause problems for docker.

Reply:
hmmm, I see `@unittest.skipUnless(os.path.exists("/run/weldr/api.socket"), "Test requires a running API server")` only in `test_blueprints.py`, not in `test_projects.py`

Reply:
You're right, I mixed the two tests I was fixing together in my head. I need to check again, but it seems that this really won't work for RHEL...

Reply:
With the changes above I think it is probably good enough.